osidb/tests/test_flaw.py

```diff
@@ -914,3 +914,29 @@ def test_validate_embargoing_public_flaw(self):
             ]
             flaw.unembargo_dt = tzdatetime(2022, 1, 1)
             flaw.save()
+            
+    @pytest.mark.parametrize(
+        "cwe,should_raise",
+        [
+            ("CWE-123", False),
+            ("(CWE-123)", False),
+            ("CWE-123|CWE-1234", False),
+            ("CWE-123->CWE-12324", False),
+            ("CWE-123->(CWE-12324)", False),
+            ("(CWE-123->(CWE-12324))", False),
+            ("CWE-1->(CWE-2->CWE-3->CWE-4|(CWE-5|CWE-6))", False),
+            ("CWE-1+CWE-2", True),
+            ("cwe-1->CWE-2", True),
+            ("cwe-1|CWE-2|", True),
```

Comment:
I would make it all uppercase so we really test just the trailing `|`

Reply:
Do you mean the test case itself only have one error scenario or the validation should be not case-sensitive?

Reply:
Oh I was definitely not clear here. I meant leave `("cwe-1->CWE-2", True),` as it is as it tests the uppercase letters but I would change `("cwe-1|CWE-2|", True),` to `("CWE-1|CWE-2|", True),` so it really tests the trailing `|`